tests/test_uncrustify/test.py

```diff
@@ -210,6 +231,7 @@ def build_from_args(self, args):
         self.test_expected = args.expected
         self.test_rerun_config = args.rerun_config or args.config
         self.test_rerun_expected = args.rerun_expected or args.expected
+        self.test_xfail = False
```

Comment:
It would be nice to not break `UNCRUSTIFY_SEPARATE_TESTS`...

Reply:
I don't really see the connection. I was just setting a default value and don't expect anything to fail. Otherwise, are you thinking in sending it as an arg?

Reply:
Right... IOW, it would be nice to see the needed changes to `print_as_ctest` and `add_test_arguments` also.